Assets/_BForBoss/_Utility/Scripts/Logger/Logger.cs

```diff
@@ -22,22 +31,51 @@ public static void LogFormat(string toLog, string key = "Misc", params object[]
 #endif
         }
         
-        public static void LogWarning(string toLog, string key = "Misc")
+        public static void LogWarning(string toLog, LoggerColor color = LoggerColor.Black, string key = "Misc")
         {
 #if !UNITY_EDITOR && DEBUG
-            Debug.LogWarning(toLog);
+            Debug.LogWarning(ColorizeLog(toLog, color));
```

Comment:
Within the context of this ticket. this is a good solution.
 
However, to take it a step further, I would separate `ColorizeLog` into its own variable, and then pass it into `LogWarning`. 

This way, if we want more customized Logs (like the `toLogs` would already have its own tags from outside), it'll be easier to update it.